pyspecProcScripts/third_level/process_IR.py

```diff
@@ -18,6 +19,7 @@ def select_pathway(s,pathway):
 def as_scan_nbr(d):
     '''since we need to relabel vd frequently- we make a method'''
     return d.C.setaxis('vd','#').set_units('vd','scan #')
+# this is written like you're trying to run it as a script!
```

Comment:
note this

Reply:
I'm not sure what you mean? it is a script?